README.md

```diff
@@ -84,42 +100,57 @@ export default class StepPlanet extends Steps {
 }
 ```
 
-The inheritance of the `Steps` class expose following class fields:
+The manager exposed his options to each steps, they can access it with `this.options`.
+The inheritance of the `Steps` class exposes following the class fields:
```

Comment:
_The inheritance of the `Steps` class exposes the following class fields:_

Reply:
C'est corrigé